dashboard/test/models/pd/regional_partner_contact_test.rb

```diff
@@ -156,33 +156,27 @@ class Pd::RegionalPartnerContactTest < ActiveSupport::TestCase
   end
 
   # If matched but no regional partner pms, send unmatched email
-  # TODO: When cc supported, fix test to check for Jenna cc
   test 'Matched with zero regional partner pms' do
     regional_partner = create :regional_partner, name: "partner_OH_45242"
     regional_partner.mappings.find_or_create_by!(state: 'OH')
     regional_partner.mappings.find_or_create_by!(zip_code: '45242')
 
     create :pd_regional_partner_contact, form_data: FORM_DATA.merge(MATCHED_FORM_DATA).to_json
-    first_mail = ActionMailer::Base.deliveries[0]
-    second_mail = ActionMailer::Base.deliveries[1]
+    mail = ActionMailer::Base.deliveries[0]
 
-    assert_equal ['nimisha@code.org'], first_mail.to
-    assert_equal ['jenna@code.org'], second_mail.to
-    assert_equal 'A school administrator wants to connect with Code.org', first_mail.subject
-    assert_equal ['tanya_parker@code.org'], first_mail.from
+    assert_equal ['tawny@code.org'], mail.to
+    assert_equal 'A school administrator wants to connect with Code.org', mail.subject
+    assert_equal ['tanya_parker@code.org'], mail.from
     assert_equal 3, ActionMailer::Base.deliveries.count
   end
 
-  # TODO: When cc supported, fix test to check for Jenna cc
   test 'Unmatched' do
     create :pd_regional_partner_contact, form_data: FORM_DATA.merge(MATCHED_FORM_DATA).to_json
-    first_mail = ActionMailer::Base.deliveries[0]
-    second_mail = ActionMailer::Base.deliveries[1]
+    mail = ActionMailer::Base.deliveries[0]
 
-    assert_equal ['nimisha@code.org'], first_mail.to
-    assert_equal ['jenna@code.org'], second_mail.to
-    assert_equal 'A school administrator wants to connect with Code.org', first_mail.subject
-    assert_equal ['tanya_parker@code.org'], first_mail.from
+    assert_equal ['tawny@code.org'], mail.to
+    assert_equal 'A school administrator wants to connect with Code.org', mail.subject
+    assert_equal ['tanya_parker@code.org'], mail.from
     assert_equal 3, ActionMailer::Base.deliveries.count
```

Comment:
Same thing here - I think this line should be `assert_equal 2, ActionMailer::Base.deliveries.count` or the test will fail